Doc/library/signal.rst

```diff
@@ -711,10 +711,11 @@ case, wrap your entry point to catch this exception as follows::
     if __name__ == '__main__':
         main()
 
-Do not set :const:`SIGPIPE`'s disposition to :const:`SIG_DFL`
-in order to avoid :exc:`BrokenPipeError`.  Doing that would cause
-your program to exit unexpectedly also whenever any socket connection
-is interrupted while your program is still writing to it.
+Do not set :const:`SIGPIPE`'s disposition to :const:`SIG_DFL` in
+order to avoid :exc:`BrokenPipeError`.  Doing that would also
+cause your program to exit unexpectedly whenever any socket
```

Comment:
```suggestion
order to avoid :exc:`BrokenPipeError`.  Doing that causes
your program to exit unexpectedly whenever any socket
```
More concise wording

Reply:
Ack, sorry for the delayed response. This doesn't quite feel right since I feel like this should be in the future tense? Maybe just remove "also", though I don't care _that_ much and don't want to kill a bunch of time bikeshedding.

Reply:
Sure, let's change it back to "would cause" and then I'll merge it